2-networks/modules/standard_shared_vpc/dns.tf

```diff
@@ -0,0 +1,112 @@
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/******************************************
+  Default DNS Policy
+ *****************************************/
+
+resource "google_dns_policy" "default_policy" {
+  provider                  = google-beta
+  project                   = var.project_id
+  name                      = "default-policy"
+  enable_inbound_forwarding = var.dns_enable_inbound_forwarding
+  enable_logging            = var.dns_enable_logging
+  networks {
+    network_url = module.main.network_self_link
+  }
+}
+
+/******************************************
+  Private Google APIs DNS Zone & records.
+ *****************************************/
+
+resource "google_dns_managed_zone" "private_googleapis" {
```

Comment:
Could we use the DNS module to handle this? https://github.com/terraform-google-modules/terraform-google-cloud-dns

Reply:
Sure thing, this will make it more concise